### PR TITLE
[rllib] Switch DQN to using deepmind wrappers

### PR DESCRIPTION
**Pull request by ericl**
## What do these changes do?

Follow baselines and switch to using the deepmind atari wrappers. Couple minor differences:
- We use 80x80 instead of 84x84. This is because our conv network only accepts 80x80 by default.
- We use uint8 instead of float32. This reduces memory usage and makes gradient computation much faster.
- We support non-*NoFrameskip environments -- in those cases, we don't add the MaxAndSkip wrapper.

## Related issue number

https://github.com/ray-project/ray/issues/1590

### CODE REVIEW

File: python/ray/rllib/optimizers/apex_optimizer.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def __init__(
         self.prioritized_replay_eps = prioritized_replay_eps
 
         self.replay_buffer = PrioritizedReplayBuffer(
-            buffer_size, alpha=prioritized_replay_alpha)
+            self.buffer_size, alpha=prioritized_replay_alpha)
```
Comment (ericl) [inline python/ray/rllib/optimizers/apex_optimizer.py]: Bug fix.

File: python/ray/rllib/dqn/common/atari_wrappers.py [hunk 2]
```diff
@@ -0,0 +1,209 @@
+import numpy as np
+from collections import deque
+import gym
+from gym import spaces
+import cv2
+cv2.ocl.setUseOpenCL(False)
```
Comment (richardliaw) [inline python/ray/rllib/dqn/common/atari_wrappers.py]: what does this do?

File: python/ray/rllib/dqn/common/atari_wrappers.py [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+        return frame[:, :, None]
+
+
+class FrameStack(gym.Wrapper):
+    def __init__(self, env, k):
+        """Stack k last frames.
+
+        Returns lazy array, which is much more memory efficient.
+
+        See Also
```
Comment (richardliaw) [inline python/ray/rllib/dqn/common/atari_wrappers.py]: can we fix the documentation?

**Review by richardliaw — APPROVED** [review]

File: python/ray/rllib/dqn/common/atari_wrappers.py (hunk 2 above)
Comment (ericl) [reply python/ray/rllib/dqn/common/atari_wrappers.py]: I have no idea -- this is copied from the baselines copy.

File: python/ray/rllib/dqn/common/atari_wrappers.py (hunk 3 above)
Comment (ericl) [reply python/ray/rllib/dqn/common/atari_wrappers.py]: Done

**Pull request merged**